backend/api/logger.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"log"
+	"net/http"
+	"time"
+)
+
+func Logger(inner http.Handler, name string) http.Handler {
```

Comment:
The logger is not used. When I am visiting `localhost:8080/lanets`, i am getting the LAN ETS logo but there is no logs on the console. Is this intended?